examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py

```diff
@@ -1,5 +1,9 @@
 import asyncio
 import logging
+import grpc
+
+import pyinjective.proto.exchange.injective_derivative_exchange_rpc_pb2 as derivative_exchange_rpc_pb
```

Comment:
We want to abstract away everything about proto files from users so this doesn't look good

Reply:
But realistically nobody is going to take order hashes one by one to append them in a list, they'll have to make this query to cancel all orders from a market or multiple markets given they have 5-10+ orders.

Reply:
Yes, that is covered in another example already. Each of the example demonstrate a discrete action assuming all the inputs are available. The user can compose their script base off these examples, like assembling lego pieces.

For example, deposit funds into subaccount -> fetch all the market ids -> pick the markets they want to trade -> stream the prices -> decide when to create/cancel/batch create/batch cancel/liquidate orders -> get order hashes from markets they just act upon -> continue to stream price and send msgs.